app/controllers/orgs/lti_configurations_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Orgs
+  class LtiConfigurationsController < Orgs::Controller
+    before_action :ensure_lti_launch_flipper_is_enabled
+    before_action :ensure_current_lti_configuration, except: %i[new create]
+
+    def create
+      # Create a new lti configuration for the current organization here
+      # redirect_to lti_configuration_path(current_organization) after
```

Comment:
Is this a `TODO`? Or are you documenting this method?

Reply:
This is a TO-DO. This PR just focuses on the front end of the application. The next PR will include the back-end logic.

Reply:
Typically the convention for leaving todos is to start the comment with `TODO` so that a other developers can discover the TODO's with [`rake notes`](https://guides.rubyonrails.org/command_line.html#notes)